gateserver/messagehandler.py

```diff
@@ -0,0 +1,28 @@
+import socketserver
+import psycopg2
+
+class MessageHandler(socketserver.BaseRequestHandler):
+    """Handles a message from the controller.
+
+    Behaves according to
+    https://github.com/fmfi-svt/gate/wiki/Controller-%E2%86%94-Server-Protocol .
+    """
+
+    _db_conn = None
+
+    @classmethod
+    def set_db(cls, db_conf):
+        """Connects to the database. All instances will share the connection."""
+        cls._db_conn   = psycopg2.connect(db_conf)
```

Comment:
Code style: medzery pred `=`.
Hmm. Nie je to az take dolezite, ale dalo by sa to nejako zneglobalnit? (Napr ak ma handler pristup ku serveru, mohol by to byt atribut na serveri.)